docker/release/docker-compose.yml

```diff
@@ -13,7 +12,6 @@ services:
       - ./logs:/var/log/timesketch/
 
   timesketch-worker:
```

Comment:
Not sure why the container_names should be removed in this PR. If they need to be removed I opt to make it in a dedicated PR so it is traceable

Reply:
same for the container names below

Reply:
good catch. I have reverted back.